code/game/machinery/computer/tram_controls.dm

```diff
@@ -15,14 +16,28 @@
 	//find the tram, late so the tram is all... set up so when this is called? i'm seriously stupid and 90% of what i do consists of barely educated guessing :)
 	find_tram()
 
+/obj/machinery/computer/tram_controls/Destroy()
+	for(var/obj/structure/industrial_lift/tram/tram_finder in GLOB.lifts)
```

Comment:
Is list this exclusively /tram objs? If so, use `as anything`

Reply:
the regular industrial lifts use this list as well currently